napari/_app_model/actions/_toggle_action.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from app_model.types import Action, ToggleRule
+
+if TYPE_CHECKING:
+    from ...viewer import Viewer
+
+
+class ViewerToggleAction(Action):
+    """Action subclass that toggles a boolean viewer (sub)attribute on trigger.
+
+    Parameters
+    ----------
+    id : CommandId
+        The id of the action.
+    title : str
+        The title of the action.
```

Comment:
Is this the idea that you're suggesting @lucyleeow?

```suggestion
        The title of the action. Prefer capital case.
```

Other options would be sentence case or no caps. Not sure what the preferred option is, but agree that it's nice to add a hint to the docs for consistency. However, I don't feel we should have any actual validation rule around casing.

Reply:
cc @jni

Reply:
Just saw this - also spoke to @jni yesterday and he agrees it should be titlecase as is standard on Mac, Windows & Ubuntu.